core/src/main/scala/akka/persistence/postgres/db/DbErrors.scala

```diff
@@ -26,4 +27,18 @@ object DbErrors {
         logger.debug(s"Created missing journal partition for $partitionDetails")
         DBIO.successful(())
     }
+
+  def withHandledIndexErrors(logger: Logger, indexDetails: String)(dbio: DBIOAction[_, NoStream, Effect])(implicit
+      ec: ExecutionContext): DBIOAction[Unit, NoStream, Effect] =
+    dbio.asTry.flatMap {
+      case Failure(ex: SQLException) if ex.getSQLState == PgUniqueViolation =>
```

Comment:
Can this happen, even with the `IF NOT EXISTS` ?

Reply:
absolutely, it happens even with the if not exists, I'm not sure why